packages/next/client/image.tsx

```diff
@@ -451,28 +397,17 @@ export default function Image({
   })
 
   let imgAttributes:
-    | {
-        src: string
-        srcSet?: string
-      }
-    | {
-        'data-src': string
-        'data-srcset'?: string
-      }
+    | Pick<JSX.IntrinsicElements['img'], 'src' | 'srcSet'>
+    | undefined
+
   if (!lazy) {
     imgAttributes = {
       src: imgSrc,
     }
     if (imgSrcSet) {
       imgAttributes.srcSet = imgSrcSet
     }
-  } else {
-    imgAttributes = {
-      'data-src': imgSrc,
-    }
-    if (imgSrcSet) {
-      imgAttributes['data-srcset'] = imgSrcSet
-    }
+  } else if (process.env.__NEXT_TEST_MODE) {
```

Comment:
What's the purpose of `__NEXT_TEST_MODE` here?

Reply:
To code eliminate the usage of the `__lazy` class, which I could only see used for a test:
![image](https://user-images.githubusercontent.com/616428/98417981-f7fd2600-204f-11eb-9c8e-5941fb744d91.png)

I didn't think the `__lazy` was meant to be public API?

Reply:
Yeah me and @styfle were talking about this yesterday--the __lazy class originally had another purpose (I think to hide the image before it loaded?) but got orphaned. We were discussing whether we wanted to leave it for the cases where people wanted to do custom styling while images are loading. I'm ok with either way.

Reply:
Let's remove it in favor of the `placeholder` prop which will make sure the lazy state is properly counted as LCP. A custom lazy style could hurt LCP.